db_init.py

```diff
@@ -0,0 +1,78 @@
+import sqlite3 as sql
+from pathlib import Path
+
+
+def check_and_create_db():
+    checked_directory = Path(Path.cwd(), "data")
+    checked_file = Path(Path.cwd(), "data", "data_sql.db")
+
+    if Path.exists(checked_directory):
```

Comment:
// if folder not exists
// - create folder

// if database not exist
// - create database